.buildkite/scripts/pipelines/security_solution_quality_gate/pipeline.sh

```diff
@@ -1,5 +1,16 @@
 #!/bin/bash
-
 set -euo pipefail
 
-echo "In the entrypoint for the quality gate"
\ No newline at end of file
+source .buildkite/scripts/common/util.sh
+source .buildkite/scripts/steps/functional/common_cypress.sh
+.buildkite/scripts/bootstrap.sh
+
+export JOB=kibana-security-solution-chrome
+
+buildkite-agent meta-data set "${BUILDKITE_JOB_ID}_is_test_execution_step" "true"
+
+echo "--- Serverless Security Second Quality Gate"
+cd x-pack/test/security_solution_cypress
+set +e
+
+CYPRESS_ELASTICSEARCH_URL=$TEST_ENV_ES_URL CYPRESS_BASE_URL=$TEST_ENV_KB_URL CYPRESS_ELASTICSEARCH_USERNAME=$TEST_ENV_USERNAME CYPRESS_ELASTICSEARCH_PASSWORD=$TEST_ENV_PWD CYPRESS_KIBANA_URL=$CYPRESS_BASE_URL yarn cypress:run:qa:serverless; status=$?; yarn junit:merge || :; exit $status
```

Comment:
If this is running against remote test environments, we're going to want to sanitize https://buildkite.com/elastic/kibana-pull-request/builds/164336#018aef91-5b97-46f0-90af-08d94f7bc032/268-432

Reply:
@jbudz the printed values are due to this [script](x-pack/plugins/security_solution/scripts/run_cypress/parallel.ts) that it is not used for executing tests on remote environments, so no sensitive information is going to be shared.

@dkirchan may you please double-check that your pipeline is not going to display that info? Thanks!

Reply:
@MadameSheema @jbudz Actually it does. So the flow is:
- A pipeline (security quality gate) creates a QA environment and gets its connection details and passes them over to the quality gate pipeline via [environment variables ](https://buildkite.com/elastic/kibana-serverless-security-solution-quality-gate/builds/81#018afb1e-7336-455f-bc30-23021ea73969). If you navigate to the environment variables you ll find them. 
- When the quality gate pipeline finishes the environment is destroyed. So the exposure of this variables is somehow  a "problem" as far as the kibana tests are running. I dont know if I can somehow mask the env vars in buildkite. 
- The script does not print out anything in the console output though!

Reply:
I would recommend passing an environment id and using it as a key lookup in `vault` for sensitive information.  We don't want this information to be visible.

https://github.com/elastic/kibana/blob/main/.buildkite/scripts/steps/cloud/build_and_deploy.sh#L83 has an example.

Reply:
I really thought this would be the best way forward after your message. On it to change it. @MadameSheema I m working on it.

Reply:
It looks like this is resolved, so I'll approve since Jon is out.

Reply:
Fixed with encryption key in secrets.elastic.co